### PR TITLE
fix: improve log support

### PR DESCRIPTION
**Pull request by closetool**
Signed-off-by: closetool <c299999999@qq.com>

Fix: #104

### CODE REVIEW

**Comment by closetool** [comment]
@hsluoyz @arafat-java plz review

File: adapter.go [hunk 1]
```diff
@@ -26,9 +26,11 @@ import (
 	"github.com/jackc/pgconn"
 	"gorm.io/driver/mysql"
 	"gorm.io/driver/postgres"
+
```
Comment (hsluoyz) [inline adapter.go:29]: no new line

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Comment by closetool** [comment]
@arafat-java fixed

**Comment by hsluoyz** [comment]
@closetool why there's no GitHub Actions bot? I want this: https://github.com/casbin/casbin/pull/773#issuecomment-826350808

image

**Pull request merged**